zcash_test_vectors/orchard_zsa/asset_base.py

```diff
@@ -24,6 +24,11 @@ def asset_digest(encoded_asset_id):
     h.update(encoded_asset_id)
     return h.digest()
 
+def asset_desc_digest(asset_desc):
+    h = blake2b(digest_size=32, person=b"ZSA-AssetDescCRH")
+    h.update(asset_desc)
+    return h.digest()
+
```

Comment:
the parameter names for `encode_asset_id()` should be adjusted to
`key` and `asset_desc_hash`

Reply:
the order of the functions should reflect the actual order of the spec:
```python
def asset_desc_digest(asset_desc):
    ..

def encode_asset_id(key, asset_desc_hash):
    ..

def asset_digest(encoded_asset_id):
    ..

def zsa_value_base(asset_digest_value):
    ..
```

Reply:
Done